pages/dkp/kaptain/2.0.0/install/prerequisites/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: layout.pug
+navigationTitle: Prerequisites
+title: Prerequisites for installing Kaptain
+menuWeight: 2
+excerpt: Ensure these prerequisites are met before installing Kaptain
+beta: false
+enterprise: false
+---
+
```

Comment:
fair to assume that the warning that's on the confluence page and missing here (about clusters and workspaces) does not apply to this version of DKP/Kaptain?

Reply:
No, this should be here as well! Kaptain was never intended to be installed on a workspace with multiple clusters. Adding right now...